internal/cache/metadata/stat_cache_test.go

```diff
@@ -276,3 +299,64 @@ func (t *StatCacheTest) Overwrite_NegativeWithNegative() {
 
 	ExpectTrue(t.cache.NegativeEntry(name, someTime))
 }
+
+// ///////////////////////////////////////////////////////////////
+// ////// Tests for multi-bucket cache scenarios /////////////////
+// ///////////////////////////////////////////////////////////////
+var (
+	apple    = &gcs.Object{Name: "apple"}
+	orange   = &gcs.Object{Name: "orange"}
+	cardamom = &gcs.Object{Name: "cardamom"}
+)
+
+func (t *MultiBucketStatCacheTest) FillUpToCapacityWithMultipleBuckets() {
```

Comment:
Add tests for following scenarios.
1. Adding object with same name to both fruits and spices bucket's cache views.
2. Add tests for fruitsExtended bucket.

Reply:
1st done. 2nd not needed.